code-challenges.test.js

```diff
@@ -87,8 +220,62 @@ var vowelTester2 = "academy"
 var vowelTester3 = "challenge"
 // Expected output: 2
 
+describe("When firstVow is called,", () => {
+
+    it("returns the index of the first vowel of the string 'learn'", () => {
+
+        const vowelTester1 = "learn";
 
+        const actResult = firstVow(vowelTester1);
 
+        expect(actResult).toEqual(1);
+    })
+
+    it("returns the index of the first vowel of the string 'academy'", () => {
+
+        const vowelTester1 = "academy";
+
+        const actResult = firstVow(vowelTester1);
+
+        expect(actResult).toEqual(0);
+    })
+
+    it("returns the index of the first vowel of the string 'challenge'", () => {
+
+        const vowelTester1 = "challenge";
+
+        const actResult = firstVow(vowelTester1);
+
+        expect(actResult).toEqual(2);
+    })
+})
 
 
 // b) Create the function that makes the test pass.
+
+const firstVow = (string) => {
+
+    string = string.split("");
+    let boolArr = [];
+
+    for(i = 0; i < string.length; i++){
```

Comment:
pretty much anytime a for-loop is being use to iterate through an array and push data into a new array, we should be using `.map()`